doc/nrf/libraries/modem/location.rst

```diff
@@ -260,8 +318,6 @@ Limitations
 ***********
 
 * The Location library can only have one application registered at a time. If there is already an application handler registered, another initialization will override the existing handler.
-* Cellular neighbor information used for cellular positioning is more accurate on modem firmware (MFW) 1.3.0 compared to earlier MFW releases that do not have an API for scanning the neighboring cells.
-  For MFW releases older than 1.3.0, only serving cell information is provided and it can be hours or days old, or even older, depending on the modem sleep states.
```

Comment:
I guess this statement can be removed given serving cell information is not used anymore due to other PRs.
@tokangas

Reply:
True, it can be removed.